tests/periph_eeprom/Makefile

```diff
@@ -0,0 +1,9 @@
+BOARD ?= b-l072z-lrwan1
```

Comment:
why this board? Maybe chose one that is more common?

Reply:
This is my dev board actually and it's also provided by iotlab ;)